nsqlookupd/http.go

```diff
@@ -39,6 +39,7 @@ func newHTTPServer(ctx *Context) *httpServer {
 	router.Handle("GET", "/topics", http_api.Decorate(s.doTopics, log, http_api.V1))
 	router.Handle("GET", "/channels", http_api.Decorate(s.doChannels, log, http_api.V1))
 	router.Handle("GET", "/nodes", http_api.Decorate(s.doNodes, log, http_api.V1))
+	router.Handle("GET", "/topic/meta", http_api.Decorate(s.doTopicMeta, log, http_api.V1))
```

Comment:
I'd prefer to not add an endpoint named `/topic/meta`. The best alternative I can come up with is `/topics?topic=...`